content/cpp/concepts/deque/terms/push_back/push_back.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.push_back()'
+Description: 'Adds an element to the end of the deque.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Containers'
+  - 'OOP'
+  - 'Classes'
+  - 'Deques'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`deque.push_back()`** method adds an element to the end of the `deque`.
+
+## Syntax
+
+The `push_back()` method is called on a `deque` using the following syntax:
+
+```pseudo
+dequeName.push_back(value);
+```
+
```

Comment:
```suggestion

- `value`: The element to be added to the back of the deque. It can be of any data type that the deque holds.

```

Reply:
The parameter needs to be described, you can change the description, this is just for example